mrpt_reactivenav2d/src/mrpt_reactivenav2d_node.cpp

```diff
@@ -185,8 +192,10 @@ class ReactiveNav2DNode
 #if MRPT_VERSION>=0x150
 		bool stop()
 		{
-			std::vector<double> cmds(2,0.0);
-			return changeSpeeds(cmds);
+			mrpt::kinematics::CVehicleVelCmd_DiffDriven vel_cmd;
+			vel_cmd.lin_vel = 2;
```

Comment:
This should be: 
vel_cmd.lin_vel = 0;

the "2" before in `cmds(2,0.0);` was due to the length of the vector, and all of the elements set to "0.0"....
Not clear at all, hence the refactoring to the new structure in which each velocity component is referenced by name ;-)